README.md

```diff
@@ -1,308 +1,60 @@
-kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/multus/multus.yaml
-```
-
-### Prerequisites
-* [Kubevirt v0.37.1 or above](https://github.com/kubevirt/kubevirt/releases/tag/v0.37.1) must be installed on the cluster.
-* [CNI plugins](https://github.com/containernetworking/plugins) must be installed in the cluster, for example
-```bash
-kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/hack/cni-plugins/cni-plugins.yaml
-```
-
-* Download the project yaml and modify its mac range to avoid collisions with nearby clusters.
-  Finally, deploy the project by applying its yaml.
-```bash
-wget https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/config/release/kubemacpool.yaml
-mac_oui=02:`openssl rand -hex 1`:`openssl rand -hex 1`
-sed -i "s/02:00:00:00:00:00/$mac_oui:00:00:00/" kubemacpool.yaml
-sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
-kubectl apply -f ./kubemacpool.yaml
-```
-
-### kubemacpool service
-
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on Pods and VirtualMachines.
-The user can specify whether MAC address will be allocated to all VirtualMachines and Pods by default or not.
-This can be done by patching of the following snippets. Patched manifest will be on [testing manifests](config/test/kubemacpool.yaml):
-
-To set kubemacpool to allocate all VirtualMachines and Pods by default (see [opt-out Mode](#How-to-opt-out-kubemacpool-MAC-assignment-for-a-namespace-in-Opt-out-mode)):
-```bash
-cp config/default/mutatepods_opt_out_patch.yaml config/test/mutatepods_opt_mode_patch.yaml
-cp config/default/mutatevirtualmachines_opt_out_patch.yaml config/test/mutatevirtualmachines_opt_mode_patch.yaml
-make generate-test
-```
-
-To set kubemacpool to not allocate all VirtualMachines and Pods by default (see [opt-in Mode](#How-to-opt-in-kubemacpool-MAC-assignment-for-a-namespace-in-Opt-in-mode))):
-```bash
-cp config/default/mutatepods_opt_in_patch.yaml config/test/mutatepods_opt_mode_patch.yaml
-cp config/default/mutatevirtualmachines_opt_in_patch.yaml config/test/mutatevirtualmachines_opt_mode_patch.yaml
-make generate-test
-```
-
-**note:** The User can of-course set Pods and VirtualMachines to separate opt-modes by sub-setting the above snippets. 
-
-**note:** The default opt-mode for [testing manifests](config/test/kubemacpool.yaml) is opt-out for VirtualMachines and Pods. 
-
-**note:** The default opt-mode for [release manifests](config/release/kubemacpool.yaml) is opt-out for VirtualMachines and opt-out for Pods. 
-
-#### How to opt-out kubemacpool MAC assignment for a namespace in Opt-out mode
-
-You can opt-out MAC assignment on your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=ignore` - to opt-out MAC assignment for Pods in your namespace
-- `mutatevirtualmachines.kubemacpool.io=ignore` - to opt-out MAC assignment for VMs in your namespace
+## Next Steps
```

Comment:
Next steps sound like "extra". Maybe "Usage" would be better?

Reply:
Usage is better I agree. Done